### PR TITLE
BaSP-TG-143:Super Admin Profile

### PR DESCRIPTION
**Pull request by Ansitox**
- Add profile layout to super admin
- Add reset, change password and save buttons

QA:
Hot to test?:
Log in as a Super Admin (email: oclacep@wikimedia.org, password: prueba1234)
When logged you should se a nav bar (temporary), move to the profile link.
In the profile you should see all inputs with the data of the current logged user.
The button "Change password" should open a modal with two passsword inputs, to try it, write the new password on both inputs (once you ended with the tests please change the password again to "prueba1234")
To try the "Save changes" button change the password that is in the password input.
The "reset" button should reset the fields you changed.
Notes:
The nav bar is temporaryand just to use on the test of the PRs, any bad function on navigation should be ignored.
The input "Password" will be deleted, but is necessary now to test the submit button.

### CODE REVIEW

**Review by matescarabino — APPROVED** [review]
LGTM

File: src/Components/SuperAdmins/NavBar/index.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+const NavBar = () => {
+  const dispatch = useDispatch();
+  const { modalContent, showConfirmModal } = useSelector((state) => state.auth);
+  const onConfirm = () => {
+    dispatch(logout());
+    dispatch(confirmModalClose());
+  };
+
+  const onSubmit = () => {
+    const content = `Are you sure you want to logout?`;
```
Comment (CristianBucci) [inline src/Components/SuperAdmins/NavBar/index.js:17]: Change backticks!

**Review by CristianBucci — APPROVED** [review]
Everything looks right!

File: src/Components/SuperAdmins/NavBar/index.js (hunk 1 above)
Comment (Ansitox) [reply src/Components/SuperAdmins/NavBar/index.js]: done

**Review by NicoSan02 — APPROVED** [review]
🕵️‍♂️ QA FEEDBACK

We wanted to tell you that we consider that your pr complies the requirements to be approved, and it can be merged into the master branch 👍

**Review by julianpiaggio — APPROVED** [review]
🕵️‍♂️ QA FEEDBACK

First of all, we wanted to tell you that we consider that your pr complies
the requirements to be approved, and that it can be merged into the master branch 👍

Now, if we consider leaving you some comments:

There is a bug reported by the developers, when you try to change the password in the super admin profile and want to save those changes, what is saved is not what was changed, but what was there before. It was indicated that in trello ticket 153 this bug was fixed. It is left here only for informational purposes.

**Pull request merged**